### ISSUE TITLE
Netflix issue

### ISSUE THREAD

**Issue by challgren** [SoundControl/Issues#153]
I'm not sure Netflix is offically supported but when watching a Youtube video and finishing it. Soundcontrol will unpause Netflix however the GUI on Netflix doesn't update. The play button is still play instead of pause and the description of the episode will still show almost as if the show/movie is still paused.

**Comment by SoundControl** [comment]
It sounds like Netflix is experimenting with their html5 video player. This
is almost certainly happening because you have Web Audio enabled.

Netflix may be able to be added as a supported player soon.

Thanks for letting me know!

**Comment by challgren** [comment]
Yeah when I hover over sound control it says "Netflix\n by: www.netflix.com" I'm using Google Chrome Version 39.0.2171.99 m (64-bit)
